SignUp.html

```diff
@@ -0,0 +1,151 @@
+              <li> <a href="Login.html">SignIn</a> </li>
+               <li> <a href="SignUp.html">Register</a></li>
+               </ul>
+           </div>
+        </div>
+    </div>
+       <!--Top navigation Ends-->
+
+        <!--Main Navigation-->
+
+        <nav class="navi">
+        <div class="row">  
+          <div class="col span-1-of-3">
+            <a href="index.html"> <img alt="Logo" src="resources/img/logo.png" class="logo-image"> </a> </li>
+          </div>
+
+          <div class="col span-1-of-3">
+           
+                <ul class="main-nav">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="#">Products</a></li>   
+                </ul>
+         </div>
+          <div class="col span-1-of-3">
+              <div class="cart">
+            <a href="#" class="cart-link"><img src="resources/img/cart.svg" alt="cart" class="cart-img"> 0 items </a>
+         </div>
+         </div>
+        </nav>
+
+</header>
+        <!-- Main Navigation Ends-->
+<!--Login Form-->
+<div class="loginform">
+    <div class="left">
+      <h1>Signup</h1>
+      <span>We do not share your personal details with anyone.</span>
+    </div>
+    <div class="right">
+       <form method="post" name="signupform" id="signupform" autocomplete="false" action="#" onsubmit="return store(this);">
+       <div class="txt-field">
+          <input type="text" id="name"  autocomplete="false" required>
+          <span></span>
+          <label for="name">First Name</label>
+        </div>
+        <div class="txt-field">
+            <input type="text"  id="lastname" autocomplete="false" required> 
+            <span></span>
+            <label for="lastname">Last Name</label>
+          </div>
+          <div class="txt-field">
+            <input type="email" id="email" autocomplete="false" required>
+            <span></span>
+            <label for="email">Email</label>
+          </div>
+          <div class="txt-field">
+            <input type="password" id="password" autocomplete="false" pattern="^.*(?=.{6,})(?=.*[a-zA-Z])[a-zA-Z0-9]+$" title="Password must have minimum length 6 characters, a number and alphabet
+            and cannot have spaces." required>
+            <span></span>
+        
+            <label for="password">Password</label>
+          </div>
+        <div class="txt-field">
+          <input type="password" id="confirmpassword"  autocomplete="false" required>
+          <span></span>
+          <label for="confirmpassword"> Confirm Password</label>
+        </div>
+        <input type="submit" value="Signup" class="btn-login" onClick="clickedButton()"/>
+      </form>
+    </div>
+</div>
+<div class="clearfix">
+</div>
+<div style="margin-top:480px;" class="extradiv">
+
+</div>
+<!--Login Form Ends -->
+<!--Footer Section-->
+<div class="row">
+<footer class="bottom">
+    <span>  Copyright &#169; 2011-2018 Sabka Bazaar Groceries Supplies Pvt. Ltd. </span>
+  </footer>
+</div>
```

Comment:
create a template for footer as well and add it on any page needed. Handlebars provides the functionality of creating templates
https://handlebarsjs.com/guide/partials.html#inline-partials